core/domain/email_manager_test.py

```diff
@@ -2054,6 +2058,3642 @@ def test_that_correct_completion_email_is_sent(self):
+                '<p>What is the meaning of life?</p>',
+                self.mocked_review_submission_datetime))
+        reviewable_suggestion_email_info = (
+            suggestion_services
+            .create_reviewable_suggestion_email_info_from_suggestion(
+                question_suggestion))
+        review_wait_time = 5
+        mocked_datetime_for_utcnow = (
+            reviewable_suggestion_email_info.submission_datetime +
+            datetime.timedelta(days=review_wait_time))
+        expected_email_html_body = (
+            'Hi user1,'
+            '<br><br>'
+            'There are suggestions on the '
+            '<a href="%s%s">Contributor Dashboard</a> that have been waiting '
+            'for more than 0 days for review. Please take a look at the '
+            'suggestions mentioned below and help them get reviewed by going '
+            'to the <a href="%s%s#/roles">admin roles page</a> and either:'
+            '<br><br><ul>'
+            '<li>Add more reviewers to the suggestion types that have '
+            'suggestions waiting too long for a review</li><br>'
+            '<li>Find the existing reviewers and email reviewers directly '
+            'about the suggestions waiting for a review</li><br>'
+            '</ul><br>'
+            'The suggestions that have been waiting too long for a review:'
+            '<br><br>'
+            '<ul>'
+            '<li>The following question suggestion was submitted for review 5 '
+            'days ago:'
+            '<br>What is the meaning of life?</li><br>'
+            '</ul><br>'
+            'Thanks so much - we appreciate your help!<br>'
+            'Best Wishes!<br><br>'
+            '- The Oppia Contributor Dashboard Team' % (
+                feconf.OPPIA_SITE_URL, feconf.CONTRIBUTOR_DASHBOARD_URL,
+                feconf.OPPIA_SITE_URL, feconf.ADMIN_URL)
+        )
+
+        with self.can_send_emails_ctx, self.log_new_error_ctx:
+            with self.swap(
+                suggestion_models,
+                'SUGGESTION_REVIEW_WAIT_TIME_THRESHOLD_IN_DAYS', 0):
+                with self.mock_datetime_utcnow(mocked_datetime_for_utcnow):
+                    (
+                        email_manager
+                        .send_mail_to_notify_admins_suggestions_waiting_long(
+                            [self.admin_1_id],
+                            [reviewable_suggestion_email_info])
+                    )
+
+        # Make sure correct email is sent.
+        messages = self._get_sent_email_messages(self.ADMIN_1_EMAIL)
+        self.assertEqual(len(messages), 1)
+        self.assertEqual(
+            messages[0].html.decode(), expected_email_html_body)
+
+        # Make sure correct email model is stored.
+        self._assert_email_data_stored_in_sent_email_model_is_correct(
+            expected_email_html_body, self.admin_1_id,
+            self.ADMIN_1_EMAIL)
+
+    def test_email_sent_to_admin_if_question_has_waited_an_hour_for_review(
```

Comment:
I wrote all of these tests out because it was similar to the reviewer emails but then I realized we'd never email admins about suggestions that have waited minutes or seconds (or maybe even hours) for review because that's normal. Should I remove all of the tests except for the 'day' tests in this test class?